static/app/utils/getKeyCode.ts

```diff
@@ -0,0 +1,79 @@
+/* key maps and utils retrieved from  https://github.com/jaywcjlove/hotkeys */
+
+/** Includes a lot of leftover unused codes for the future in case we want glyphs for them */
+const _keyMap = {
+  backspace: 8,
+  tab: 9,
+  clear: 12,
+  enter: 13,
+  return: 13,
+  esc: 27,
+  escape: 27,
+  space: 32,
+  left: 37,
+  up: 38,
+  right: 39,
+  down: 40,
+  del: 46,
+  delete: 46,
+  ins: 45,
+  insert: 45,
+  home: 36,
+  end: 35,
+  pageup: 33,
+  pagedown: 34,
+  capslock: 20,
+  num_0: 96,
+  num_1: 97,
+  num_2: 98,
+  num_3: 99,
+  num_4: 100,
+  num_5: 101,
+  num_6: 102,
+  num_7: 103,
+  num_8: 104,
+  num_9: 105,
+  num_multiply: 106,
+  num_add: 107,
+  num_enter: 108,
+  num_subtract: 109,
+  num_decimal: 110,
+  num_divide: 111,
+  '⇪': 20,
+  ',': 188,
+  '.': 190,
+  '/': 191,
+  '`': 192,
+  '-': 189,
+  '=': 187,
+  ';': 186,
+  "'": 222,
+  '[': 219,
+  ']': 221,
+  '\\': 220,
+
+  // special case for escaped +, not actually a keycode
```

Comment:
What exactly is this for?

Reply:
This was done originally because I thought + isn't really a key you could press but wanted to render it as a +...now that I realize it does have a keyCode because you can press it on the numpad.